packages/vega4-extension/src/index.ts

```diff
@@ -120,29 +119,29 @@ export
 const rendererFactory: IRenderMime.IRendererFactory = {
   safe: true,
   mimeTypes: [VEGA_MIME_TYPE, VEGALITE_MIME_TYPE],
-  createRenderer: options => new RenderedVega3(options)
+  createRenderer: options => new RenderedVega(options)
 };
 
 const extension: IRenderMime.IExtension = {
-  id: '@jupyterlab/vega3-extension:factory',
+  id: '@jupyterlab/vega-extension:factory',
   rendererFactory,
   rank: 50,  // prefer over vega 2 extension
   dataType: 'json',
   documentWidgetFactoryOptions: [{
-    name: 'Vega 3',
-    primaryFileType: 'vega3',
-    fileTypes: ['vega3', 'json'],
-    defaultFor: ['vega3']
+    name: 'Vega',
```

Comment:
I wonder if these should still be `'Vega 4'` and `'Vega-Lite 2'` , in order to not have a name clash if a user wants to install multiple versions on their distribution.

Reply:
I suppose that's fair. If you feel this is ready, I can publish the current version.

Reply:
From my side, it's ready to go.